src/script/calling/Call.ts

```diff
@@ -90,8 +90,6 @@ export class Call {
 
     // Get the corresponding participants for the entries in ActiveSpeakers in the incoming order.
     const activeSpeakers = audio_levels
```

Comment:
Can we rename `audio_levels` to `activeSpeakers` when importing or doesn't it make sense?